projects/portal/src/app/pipes/unit-conversion.pipe.spec.ts

```diff
@@ -0,0 +1,29 @@
+import { UnitConversionPipe } from './unit-conversion.pipe';
+
+describe('UnitConversionPipe', () => {
+  const pipe = new UnitConversionPipe();
+
+  it('transform string number to K units', () => {
```

Comment:
A simple test case makes it easy to understand the movement of the target.

For example, `it transform number to string number`.

Reply:
I fix test and comment.